docker-storage-setup.1

```diff
@@ -1,10 +1,23 @@
 .TH "DOCKER-STORAGE-SETUP" "1" "NOVEMBER 2014" "Helper Script for Docker Storage Setup" ""
 .SH NAME
 .PP
-docker\-storage\-setup - Grows the root filesystem and sets up storage for container runtimes.
+docker\-storage\-setup - Tool for setting up storage for container runtimes.
 .SH SYNOPSIS
 .PP
 \f[B]docker-storage-setup\f[] [OPTIONS]
+\f[B]docker-storage-setup\f[] [OPTIONS] INPUTFILE OUTPUTFILE
```

Comment:
As of now INPUTFILE and OUTPUTFILE are optional so require brackets around these?